src/Marten.PLv8/Marten.PLv8.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <Description>Document transforms and patching extension for Marten</Description>
-        <VersionPrefix>6.0.0-alpha.3</VersionPrefix>
+        <VersionPrefix>6.0.0-alpha.4</VersionPrefix>
```

Comment:
The same.

Reply:
as above: https://github.com/JasperFx/marten/pull/2394#discussion_r1034744665